build.gradle

```diff
@@ -50,6 +50,7 @@ jacoco {
 final htsjdkVersion = System.getProperty('htsjdk.version', '2.10.1')
 
 dependencies {
+    compile 'com.intel.gkl:gkl:0.5.2'
```

Comment:
<img width="858" alt="screen shot 2017-07-05 at 9 43 04 am" src="https://user-images.githubusercontent.com/2755881/27867014-70ea1c64-6166-11e7-8806-9bdcb2463c62.png">
Confirmed! Is this something that would be good to assert on in our jar-building Gradle tasks?